README.md

```diff
@@ -199,6 +199,7 @@ maintained by [Parity Technologies](https://www.parity.io/). Source code availab
 - [SubQuery](https://subquery.network) - A GraphQL indexer and query service that allows users to easily create indexed data sources and host them online for free.
   - [Nova SubQuery API](https://github.com/nova-wallet/subquery-nova) - A SubQuery API implementation for operation history and staking analytics.
 - [Subscan](https://www.subscan.io/) - Multi-network explorer for Substrate-based chains.
+- [Subspace](https://github.com/subspace/subspace) - The first layer-one blockchain that can fully resolve the blockchain trilemma by simultaneously achieving scalability, security, and decentralization.
```

Comment:
It's certainly not the first.

Reply:
Curious what other project can you're referring to that solves the scalability issues with no compromises?

Reply:
Polkadot.  Mina maybe first.  Many zk roll ups.  (all somewhat conventional looking threat models)

Omniledger paper was kinda early too.  An implementation even exists, unsure if ever deployed.  (80% honest)

All the false cases poses a problem here too..

Optimistic roll ups claim this.  Imho they're wrong, but arguing that gets nasty:   Alone, the word "secure" means "in threat models reasonable experts accept", not "in the threat model that favors my pet project."  You need to write down their implicit assumptions before arguing they're unreaosnable.  Afaik nobody ever bothered.

Payment channels claim this.  Afaik they're secure, but not really scalable.  Although easier, arguing this still becomes nasty.

Individual bridges between two chains cannot be called scalable, but if both chains have real finality then their threat model is 2/3 honest on both, not all necessarily so bad.  A bridge ecosystem like cosmos becomes scalable, but wrecks this threat model.  It's plauslbe however that cosmos' airdrop culture kinda adequately addressed this in practice, if only by dumb luck.  Afak nobody ever surveyed zones' validator overlap.  At least this should fall appart easily if you keep everything in theory land.